packaging/hudi-presto-bundle/pom.xml

```diff
@@ -210,6 +210,7 @@
     <dependency>
       <groupId>org.apache.avro</groupId>
       <artifactId>avro</artifactId>
+      <version>1.11.3</version>
```

Comment:
If you intend to upgrade the dependency version, the upgrade should be done in `hudi-common` or root POM based on a new profile.

Reply:
@yihua We were planning to make the changes on hudi-presto-bundle only.